### PR TITLE
Add web with NextJS

### PR DESCRIPTION
**Pull request by SergioMorchon**
Just as a starter, no relevan content there except a landing page with placeholder text & apps links.

image

### CODE REVIEW

File: packages/ui/theme.css [hunk 1]
```diff
@@ -5,12 +5,14 @@ http://ionicframework.com/docs/theming/ */
 :root {
     --text-primary: #000000;
     --text-secondary: #92949c;
+    --background: #f6f6f6;
```
Comment (andyindahouse) [inline packages/ui/theme.css:8]: why do we need this?  😲

File: packages/ui/theme.css [hunk 2]
```diff
@@ -5,12 +5,14 @@ http://ionicframework.com/docs/theming/ */
...
 }
 
 @media (prefers-color-scheme: dark) {
     :root {
         --text-primary: #ffffff;
         --text-secondary: #92949c;
+        --background: #333333;
```
Comment (andyindahouse) [inline packages/ui/theme.css:15]: and this

File: packages/ui/theme.css (hunk 1 above)
Comment (SergioMorchon) [reply packages/ui/theme.css:8]: To support light & dark theme. `@kiwi/ui` texts has that feature by default, so when I was testing locally this, I found that the texts were detecting the dark theme I have on my OS, rendering themselves as white. But the background was white by default, too! xD
So I've added this variable to make it themable, and used it for the `body` background of the `next` pages.

**Review by andyindahouse — APPROVED** [review]
🚀

**Pull request merged**